setup.py

```diff
@@ -63,24 +62,27 @@ def find_version(*parts):
               'karabo-bridge >=0.6',
               'psutil',
           ],
+          'complete': [
+              'cloudpickle',
```

Comment:
I don't think cloudpickle is used for anything apart from some tests - though it's a dependency of dask, so it doesn't make much difference in practice.